changelogs/unreleased/7247-raise-warning-jit.yml

```
@@ -0,0 +1,5 @@
+description: Raise a warning if JIT is enabled on the psql db as this will degrade performances.
```

We know it had some issues on older PostgreSQL versions, but it might be better on newer versions.
```suggestion
description: Raise a warning if JIT is enabled on the psql db as this might degrade performance.
```